public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -500,7 +502,9 @@ export class GraphiteDatasource
     return this.templateSrv.containsTemplate(target.target ?? '');
   }
 
-  translateTime(date: any, roundUp?: boolean, timezone?: TimeZone) {
+  translateTime(date: DateTime | string, roundUp?: boolean, timezone?: TimeZone) {
+    let dateCopy: Moment;
```

Comment:
Mutation of `let`. IMO, it would be preferable to have a function that returned `Moment` with this logic encapsulated inside of it.